raphtory/src/db/graph_immutable.rs

```diff
@@ -16,21 +16,23 @@
 
```

Comment:
there is an argument that this could perhaps be deleted (the entire thing)

Reply:
Once we have the locked graph working yes